modules/signal/include/opencv2/signal.hpp

```diff
@@ -0,0 +1,10 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#ifndef OPENCV_SIGNAL_HPP
+#define OPENCV_SIGNAL_HPP
+
```

Comment:
Please add module declaration for documentation: http://pullrequest.opencv.org/buildbot/export/pr_contrib/3613/docs/

```
/**
@defgroup signal Signal Processing

This module includes signal processing algorithms.

@}
*/

```

Reply:
Added declaration